src/Phoenix.elm

```diff
@@ -257,29 +257,29 @@ sendPushsHelp cmds state =
                     <&> sendPushsHelp rest
 
 
-handleSocketsUpdate : Platform.Router msg (Msg msg) -> Dict String Socket -> InternalSocketsDict -> Task Never InternalSocketsDict
+handleSocketsUpdate : Platform.Router msg (Msg msg) -> Dict String (Socket msg) -> InternalSocketsDict msg -> Task Never (InternalSocketsDict msg)
```

Comment:
I did a couple of renames here that helped me understand what was going on. Happy to revert them though if you prefer the original names.